cmd/logs_test.go

```diff
@@ -1,16 +1,33 @@
 package cmd
 
 import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
 	"testing"
 
+	"github.com/astronomer/astro-cli/houston"
 	testUtil "github.com/astronomer/astro-cli/pkg/testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDeploymentLogsRootCommand(t *testing.T) {
 	testUtil.InitTestConfig()
-	output, err := executeCommand("deployment", "logs")
+	okResponse := `{
+		"data": {
+		  "appConfig": {"triggererEnabled": true, "featureFlags": { "triggererEnabled": true}}
```

Comment:
Maybe you could also add a test with the triggererEnabled = false to test that the command is skipped if the feature is disabled

Reply:
test case added